packages/@aws-cdk/aws-glue/test/test.table.ts

```diff
@@ -223,6 +223,30 @@ export = {
     test.done();
   },
 
+  'table.node.defaultChild'(test: Test) {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const database = new glue.Database(stack, 'Database', {
+      databaseName: 'database',
+    });
+
+    // WHEN
+    const table = new glue.Table(stack, 'Table', {
+      database,
+      tableName: 'table',
+      columns: [{
+        name: 'col',
+        type: glue.Schema.STRING
+      }],
+      compressed: true,
+      dataFormat: glue.DataFormat.Json,
+    });
+
+    // THEN
+    test.notEqual(table.node.defaultChild, undefined);
```

Comment:
Perhaps something like `test.ok(table.node.defaultChild instanceof CfnTable)`, no?

Reply:
Yeah, it's something I found in another package. Should I change those too?

https://github.com/aws/aws-cdk/blob/4a113e62ff9038b159a40ac4edcd2543ce24e1e9/packages/%40aws-cdk/aws-ec2/test/test.vpc.ts#L593

https://github.com/aws/aws-cdk/blob/48c26c2f59cb8c30a592af98a367f59d0995f5f3/packages/%40aws-cdk/aws-autoscaling/test/test.auto-scaling-group.ts#L696

Reply:
I created a separate PR for those other two: #4199